Notepad.Extensions.Logging/NotepadProviderOptions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Extensions.Logging
+{
+    public class NotepadProviderOptions
+    {
+        /// <summary>
+        /// Default options for <see cref="NotepadLoggerProvider"/>.
+        /// </summary>
+        public static NotepadProviderOptions Default { get; } = new NotepadProviderOptions
+        {
+            WindowName = "Untitled - Notepad",
+        };
```

Comment:
This likely doesn’t do what you expect it to do. This will essentially just help establishing a singleton instance for the options which is then shared globally. And you *mutate* that instance when you call `optionsConfiguration?.Invoke(options);` in the extension method.

A better way would be to just assign this default directly to `WindowName` and create a new instance in the `AddNotepad` extension method. Or use the options pattern and don’t worry about instantiation.

Reply:
Yes, it was my mistake, I just hurried up to create PR 😄

Reply:
It's outdated. Now supports default behaviour without window name.